src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.TimeOfFlight.Models.LidarLiteV3;
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.TimeOfFlight
+{
+    /// <summary>
+    /// Lidar Lite v3 is a long-range fixed position distance sensor by Garmin.
+    /// </summary>
+    public class LidarLiteV3 : IDisposable
+    {
+        /// <summary>
+        /// Default address for LidarLiteV3
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x62;
+
+        private GpioController _gpioController;
+        private I2cDevice _i2cDevice;
+        private int? _powerEnablePin;
+
+        /// <summary>
+        /// Initialize the LidarLiteV3
+        /// </summary>
+        /// <param name="i2cDevice">I2C device</param>
+        /// <param name="gpioController">GPIO controller</param>
+        /// <param name="powerEnablePin">The pin number used to control power to the device</param>
+        public LidarLiteV3(I2cDevice i2cDevice, GpioController gpioController = null, int? powerEnablePin = null)
+        {
+            _gpioController = gpioController;
```

Comment:
we usually do:
`_gpioController = gpioController ?? new GpioController();` and remove all the null checks later (except in Dispose method)

Reply:
will fix